scripts/01_checkpoints_to_stats/compute_stats.py

```diff
@@ -0,0 +1,49 @@
+from transformers import (
+    AutoModelForMaskedLM,
+    # AutoModelForSequenceClassification,
+    BertForSequenceClassification,
+    AutoTokenizer,
+    DataCollatorForLanguageModeling,
+)
+from datasets import load_dataset, load_from_disk, Dataset
+
+from torch.utils.data import DataLoader
+import os
+from src.metrics import get_metrics_hf_transformer, get_lm_loss_hf_transformer
+from tqdm import tqdm
+import json
+import argparse
+import glob
+
+
+def write_stats_to_file(model, out_dir, epoch, train_dataloader=None, device="cuda"):
+
+    data = get_metrics_hf_transformer(model)
+
+    # make directory if it does not exist
+    if not os.path.exists(out_dir):
+        os.makedirs(out_dir)
+
+    with open(os.path.join(out_dir, f"stats_{epoch}_losses"), "w") as f:
+        print(f)
```

Comment:
Use the `logging` library instead of printing - can simply use `logging.info(...)` for this. (This has plenty of advantages beyond just best practices - like logging while the job is still running, rather than waiting until stdout flush at the end of the job like `print` does.)